pkg/drive/event.go

```diff
@@ -174,8 +178,8 @@ func (handler *driveEventHandler) unmountDrive(ctx context.Context, drive *types
 	if len(devices) > 1 {
 		return fmt.Errorf("multiple devices %v are mounted for FSUUID %v", devices, drive.Status.FSUUID)
 	}
-	mountpoints := deviceMap[devices[0]]
-	for _, mountPoint := range mountpoints {
+	mountpoints := deviceMap[devices.ToSlice()[0]]
```

Comment:
```go
mountpoints := deviceMap[devices[0]]
```

would also work here right?

Reply:
No. Because it returns `map[string]struct{}`